### DISCUSSION TITLE
Trouble with NuxtLink

### DISCUSSION THREAD

**Discussion by Zaklo** [nuxt/nuxt#17847] [Questions] [answered]
Hey ! 

I'm having some troubles with the NuxtLink component.
When I'm on `page/myPage`, the <a> class is `router-link-active router-link-exact-active` as expected.

But when I'm on `page/myPage/[slug]`, I expect that `router-link-active` should be active, but there's no class triggered.

Is someone here having the same issue ? 

Thanks :)

**Comment by danielroe** [comment, marked as answer]
This comes from a change in Vue Router. See https://router.vuejs.org/guide/migration/#removal-of-the-exact-prop-in-router-link  and https://github.com/vuejs/rfcs/blob/master/active-rfcs/0028-router-active-link.md#summary.

**Reply by Zaklo** [reply]
Thank you @danielroe !
